virtualizarr/zarr.py

```diff
@@ -12,8 +12,7 @@
 import ujson  # type: ignore
 import xarray as xr
 from pydantic import BaseModel, ConfigDict, field_validator
-
-from virtualizarr.vendor.zarr.utils import json_dumps
+from zarr.v2.util import json_dumps
```

Comment:
I would avoid this import if possible. We are likely to remove the `v2` namespace before the `3.0` release.

Reply:
I'll revert and continue to vendor this bit of code.